app/app.js

```diff
@@ -0,0 +1,54 @@
+(function(){
+	var app = angular.module('diplomacy', []);
+	
+	var server = 'http://diplomacy2.asilika.com:9494';
+	
+	app.controller('GamesController', ['$http', function($http){
+   
+		this.games = "Hello, these are the games";
+		var gamesCtrl = this;
+		
+		$http.get( server + '/api/rest/games').success(function(response){
+		  gamesCtrl.games = response.data.data;
+		});
+	}]);
+ 
+	app.controller('MatchesController', ['$http', function($http){
+  
+		this.matches = "Hello, these are the matches";
+		var matchesCtrl = this;
+		
+		$http.get( server + '/api/rest/matches').success(function(response){
+		  matchesCtrl.matches = response.data.data;
+		});
+		
+		this.setMatch = function(matchId){
+			this.selectedMatch=this.matches[matchId];
+			
+			//Get empires for this match
+			$http.get( server + '/api/rest/matches/'+matchId).success(function(response){
+				matchesCtrl.empires = response.data.data;
+			});
+		}
+		
+		this.setEmpire = function(matchId, empireId){
+			this.selectedEmpire=this.empires[empireId];
+			
+			//Get territories for this match
+			$http.get( server + '/api/rest/matches/'+matchId+'/empires/'+empireId+'/territories?include_neighbours=1').success(function(response){
```

Comment:
I'll show you the data service pattern soon, it's pretty cool, it allows you to wrap these in a consistent way that also handles errors.

Also, it'll take care of whether you're doing `POST`, `GET`, `PUT`, `DELETE`, 'OPTIONS`, and formulate the data.